docs/concepts/esql.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[esql]]
+=== ES|QL
+
+preview::[]
+
+The Elasticsearch Query Language, {esql}, has been created to make exploring your data faster and easier using the **Discover** application. {esql} was designed to combine the 9 other languages that can be used in {es} (KQL, SQL, EQL, DSL Lucene, Painless, Canvas, Timelion, and Vega). From version 8.11 you can try this new feature, which is enabled by default. 
```

Comment:
```suggestion
The Elasticsearch Query Language, {esql}, has been created to make exploring your data faster and easier using the **Discover** application. From version 8.11 you can try this new feature, which is enabled by default. 
```
Is this part of the messaging? I haven't seen it before. If not, I would not go there.